nucliadb/nucliadb/common/datamanagers/vectorsets.py

```diff
@@ -41,7 +48,15 @@ async def exists(txn, *, kbid: str, vectorset_id: str) -> bool:
     return _find_vectorset(kb_vectorsets, vectorset_id) is not None
 
 
-async def set(txn, *, kbid: str, config: knowledgebox_pb2.VectorSetConfig):
+async def iter(
+    txn: Transaction, *, kbid: str
+) -> AsyncIterator[knowledgebox_pb2.VectorSetConfig]:
+    kb_vectorsets = await _get_or_default(txn, kbid=kbid)
```

Comment:
if you're adding the migration, can the `get_or_default` be removed now? or should still be kept?

Reply:
I prefer to keep it until everything is migrated, just in case. Afterwards we can remove it